src/libm/sleefsimdsp.c

```diff
@@ -1770,7 +1776,7 @@ EXPORT CONST VECTOR_CC vfloat xatanf(vfloat d) {
   t = vreinterpret_vf_vm(vxor_vm_vm_vm(vand_vm_vo32_vm(veq_vo_vi2_vi2(vand_vi2_vi2_vi2(q, vcast_vi2_i(2)), vcast_vi2_i(2)), vreinterpret_vm_vf(vcast_vf_f(-0.0f))), vreinterpret_vm_vf(t)));
 
 #if defined(ENABLE_NEON32) || defined(ENABLE_NEON32VFPV4)
-  t = vsel_vf_vo_vf_vf(visinf_vo_vf(d), vmulsign_vf_vf_vf(vcast_vf_f(1.5874010519681994747517056f), d), t);
+  t = vsel_vf_vo_vf_vf(visinf_vo_vf(d), vmulsign_vf_vf_vf(vcast_vf_f(1.570796326794896557998982), d), t);
```

Comment:
Just a comment that in general I would recommend using hexfloat representation for constants, that would spare a conversion from double to float, or even from the literal to floating point.

Not asking to change now but that is smth we fix library-wide. That will also avoid carrying some many decimals, which is prone to errors.

Defining constants via macros could also prevent this type of errors.

Reply:
I agree that hexadecimal is better, but I was wondering which is better this time because of the MSVC case.
I will use hexadecimal next time.